test/types/enum/diten/paramEnumOutOfBounds2.good

```diff
@@ -0,0 +1,2 @@
+paramEnumOutOfBounds2.chpl:3: error: enum cast out of bounds
+paramEnumOutOfBounds2.chpl:3: error: illegal cast from int(64) to enumt
```

Comment:
This error message seems a bit misleading. The issue is that the enum is abstract, so it doesn't support casts from integers, rather than that the integral value was too small or too great.

Reply:
sure, that sounds fine